chunks/modelfreemodels.ipynb

```diff
@@ -229,11 +229,11 @@
     "\n",
     "When might we use these?\n",
     "\n",
-    "* If the data being modelled really are suspected to consist of multiple populations\n",
+    "* If the data being modeled really are suspected to have come from multiple origins\n",
     "\n",
     "e.g.\n",
-    "- supernova luminosities (without spectroscopic typing)\n",
-    "- source vs. background photon energies"
+    "- supernova luminosities $L$ (without spectroscopic typing $T$), conditional (prior) PDF $P(L|T)$\n",
+    "- source vs. background photon energies $E$, sampling distribution $P(N|T)$, $P(E|T)$"
```

Comment:
I don't understand what the two distributions are supposed to be here. But also I may have not been very clear to begin with. I simply meant that we model observed spectra as the sum of a source and a background. P(E|T) for a single photon makes sense in that context, but P(N|T) doesn't because the actual data are total counts in each channel.